cmd/migrate/transformations/transformer.go

```diff
@@ -77,6 +86,13 @@ func (ht *HCLTransformer) TransformFile(inputPath, outputPath string) error {
 
 	// Write the transformed content with formatting
 	output := hclwrite.Format(ht.file.Bytes())
+	
+	// Post-process to fix double dollar signs and heredoc issues in cloudflare_ruleset expressions
+	// Only apply if we're dealing with heredoc expressions that need it
+	if strings.Contains(string(output), "<<-EOF") || strings.Contains(string(output), "<<-EOT") {
+		output = fixCloudflareRulesetDoubleDollarSigns(output)
```

Comment:
It's kind of a bummer we have to operate on raw bytes instead of being able to use the HCL tooling we have access to just a few lines up. Are there no settings or options we can define to that `hclwrite` understands to control the escaping in heredocs?